crates/eth-rpc/tests/kkrt_prep.rs

```diff
@@ -0,0 +1,377 @@
+use std::collections::HashMap;
```

Comment:
this file should probably not be called kkrt_prep 🥕
inspiration:
script_utils 
utils
helpers
deploy_helpers
tx_helpers